python/sglang/srt/managers/tokenizer_manager.py

```diff
@@ -657,6 +657,8 @@ async def handle_loop(self):
                         out_dict = {
                             "text": recv_obj.output_strs[i],
                             "meta_info": meta_info,
+                            "input_ids": recv_obj.origin_input_ids[i],
+                            "output_ids": recv_obj.output_ids[i],
```

Comment:
by default, we should not return this because this introduce extra overhead.

Reply:
Yes. We can keep them disabled by default to avoid extra overhead, and just add a startup flag to enable returning these IDs when needed (like in OpenRLHF scenarios).